lib/features/add_bedroom/view/add_bedroom_counter.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:flutter/material.dart';
+import 'package:hatspace/gen/assets.gen.dart';
+import 'package:hatspace/theme/widgets/hs_buttons.dart';
+
+class AddBedroomCounter extends StatelessWidget {
+  final ValueNotifier<int>? counter;
+  final String? text;
+  const AddBedroomCounter(
+      {super.key, required this.counter, required this.text});
```

Comment:
why is `counter` nullable? why is `text` nullable? they're required field, so if they need to be nullable, there must be a usecase where null value is used. What is that usecase?

Reply:
That's my bad. I was sort of blindly following old code for the buttons and just did it this way